From 177df7658cbd8c78d5f3cf043e78d02880945b85 Mon Sep 17 00:00:00 2001
From: Artem Senichev <a.senichev@yadro.com>
Date: Tue, 13 Mar 2018 15:50:51 +0300
Subject: [PATCH] Set IBM instead of AMI for eSEL

Originally created by MSI (S188).
The root cause is that the latest openbmc replace the NETFUN_AMI with
NETFUN_IBM, so bmc can't log the eSEL from PNOR.

Signed-off-by: Artem Senichev <a.senichev@yadro.com>
---
 src/include/usr/ipmi/ipmiif.H | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/include/usr/ipmi/ipmiif.H b/src/include/usr/ipmi/ipmiif.H
index 44283cc0b..0f1309e31 100644
--- a/src/include/usr/ipmi/ipmiif.H
+++ b/src/include/usr/ipmi/ipmiif.H
@@ -258,7 +258,7 @@ namespace IPMI
 
     //AMI-specific storage messages
     inline const command_t partial_add_esel(void)
-    { return std::make_pair(NETFUN_AMI, 0xf0); }
+    { return std::make_pair(NETFUN_IBM, 0xf0); }
 
     // event messages
     inline const command_t platform_event(void)
-- 
2.14.1

